k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientPresenter.kt

```diff
@@ -843,6 +845,17 @@ class RecipientPresenter(
         }
     }
 
+    fun shouldDisplayInvitationFeedback(): Boolean =
+        preferences.storage.getBoolean(SHOULD_SHOW_PLANCK_INVITE_SENT_FEEDBACK, true)
```

Comment:
so this is the feature toggle for this invite sent feedback

Reply:
yep, it is saved in disk after user selects the checkbox.